<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SKT1112049PN002" name="NUGUAPP_002_연결.연결_06_NUGU_albertAI" tests="18" failures="0" timestamp="12 Oct 2020 06:19:41 GMT" time="119.256" errors="1">
  <testcase name="TC_연결_284" time="3.975" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_285" time="1.874" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_289" time="2.381" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_281" time="29.159" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_293" time="3.506" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_297" time="5.441" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_308" time="2.066" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_295" time="3.696" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_282" time="2.745" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_286_form_288" time="10.983" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_307" time="1.486" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_296" time="2.890" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_283" time="3.359" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_310_form_311" time="14.026" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_294" time="7.090" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_292" time="3.111" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_291" time="5.230" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI"/>
  <testcase name="TC_연결_309" time="16.238" classname="NUGUAPP_002_연결.연결_06_NUGU_albertAI">
    <error type="org.testng.SkipException" message="연결 가능한 디바이스 있음. Test Skip">
      <![CDATA[org.testng.SkipException: 연결 가능한 디바이스 있음. Test Skip
	at NUGUAPP_002_연결.연결_06_NUGU_albertAI.TC_연결_309(연결_06_NUGU_albertAI.java:350)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- TC_연결_309 -->
</testsuite> <!-- NUGUAPP_002_연결.연결_06_NUGU_albertAI -->
